models/User.go

```diff
@@ -828,13 +851,15 @@ func PlaceAskOrder(userId uint32, ask *Ask) (uint32, error) {
 	l.Debugf("Check3: Passed. Creating Ask.")
 
 	oldCash := user.Cash
+	oldReservedCash := user.ReservedCash
```

Comment:
Why is this needed? You're not modifying the reservedCash anywhere in this function